Mod0/TechnicalDay2.md

```diff
@@ -0,0 +1,38 @@
+
+As humans, we often interact with physical objects in the world; let's call them `things`.  We use ourknowledge and experience to work with many different types of things with no issue.  We know to sit on a chair, and lay on a bed; even though these things are both furniture, we know that they have different uses.
+
+We also use non-tangible objects like vocabulary, numbers, and dates which allow us to communicate with each other; let's call these objects `data`.  We can use data to communicate our needs, perform math and statistics, and relate spans of time (and many other tasks!). Using data, we can understand sentences like: On Tuesday, my son will be 12 years old. He is getting 2 lego sets for his birthday.
+
+With objects and data, we have everything we need to accomplish our daily tasks.  But there is one more concept that humans rely on: lists.  We use lists to help organize all of the objects and data we know about.  Let's imagine we are moving into a new home.  We know that our clothes and furniture need to be moved.  From this, we can infer that multiple pieces of clothing and multiple pieces of furniture exist and must be moved; we use the idea of `lists` instead of explicitly stating every single item that needs to be moved.
+
+Lists also help us accomplish similar tasks more efficiently.  Let's say I need groceries.  I _could_ go to the grocery store, pick up milk, come home, put the milk away, go to the grocery store, pick up bread, come home, put the bread away, go to the gocery store, pick up coffee, come home, put the coffee away, etc...  While this method would work, it is not very efficient; it is much better to create a list of all the items I need from the grocery store, then use that list at the store to pick up everything I need.
+
+
+### Things and Computers
+
+Unlike humans, computers don't know about very many things or data; but, they do know a few types of things and a few types of data. And, they can store lists!
+
+Work through the tutorial linked below to learn about the things and data that c# knows.  Make sure to walk through each `Try it yourself` section.
+* [Built in Types](https://www.w3schools.com/cs/cs_data_types.php)
+
+Work through the tutorials below to learn about how c# manages lists, and uses lists to accomplish specific tasks. Make sure to walk through each `Try it yourself` section.
+* [Arrays](https://www.w3schools.com/cs/cs_arrays.php)
+* [Looping](https://www.w3schools.com/cs/cs_arrays_loop.php)
+
+## Putting it all together
+
+Open [this repl](https://replit.com/@MeganMcMahon1/WorkingWithLists#main.cs) and complete the following:
+1. Fork the repl so that you have your own copy.
+2. Identify as many of the tools you learned about above as you can.  Make a [comment](https://www.w3schools.com/cs/cs_comments.php) above each line where you see something you recognize. You may not recognize everything in this code - that's ok!
+3. Run the program by clicking on the green play button!
```

Comment:
Thoughts on moving run the program to step 2? Then comment as step 3? Feels maybe easier to understand once they have seen what it does?

Reply:
yes - totally makes more sense :)